source/blog/2022-03-08-announcing-hanami-200alpha7.html.markdown

```diff
@@ -0,0 +1,160 @@
+---
+title: Announcing Hanami v2.0.0.alpha7
+date: 2022-03-08 12:00:00 UTC
+tags: announcements
+author: Tim Riley
+image: true
+excerpt: >
+  Concrete slice classes, consistent action and view class structures
+---
+
+Hi again, Hanami friends, we’re happy to be back this month to share the release of Hanami 2.0.0.alpha7.
+
+Before we go further, we want to make it clear that **we stand with the people of Ukraine in the face of the senseless, horrific attacks on their homeland by the Russian military.**
+
+If you can, [**please donate to Ukraine’s defense**](https://war.ukraine.ua), or show your support in any way possible.
+
+At very least, [read this perspective](https://zverok.space/blog/2022-03-03-WAR.html) from our Ruby friend Victor “zverok” Shepelev, who is currently sheltering in the heavily bombarded Kharkiv, Ukraine with his family. Victor should be spreading his Ruby magic, not dodging missiles.
+
+As for this Hanami alpha release: it contains improvements to slice configuration, as well as the action and view class structure within generated applications.
+
+## Slice configuration via concrete classes
+
+You can now configure your slices by creating concrete classes for each slice in `config/slices/`:
+
+```ruby
+# config/slices/main.rb:
+
+module Main
+  class Slice < Hanami::Slice
+    # slice config here
+  end
+end
+```
+
+The application-level `config.slice(slice_name, &block)` setting has been removed in favour of these classes, which provide much a clearer place for configuration.
+
+As of this release, you can use the slice classes to configure your slice imports:
+
+```ruby
+# config/slices/main.rb:
+
+module Main
+  class Slice < Hanami::Slice
+    # Import all exported components from "search" slice
+    import from: :search
+  end
+end
+```
+
+As well as particular components to export:
+
+```ruby
+# config/slices/search.rb:
+
+module Search
+  class Slice < Hanami::Slice
+    # Export the "index_entity" component only
+    export ["index_entity"]
+  end
+end
+```
+
+Hanami slices use dry-system to manage their internal container of components, and it does all the work to configure this container to work according to the Hanami application conventions as well as your own specific application configuration. However, for advanced cases, you can also directly configure the slice’s container using `prepare_container` on the slice class:
```

Comment:
```suggestion
Hanami slices use `dry-system` to manage their internal container of components, and it does all the work to configure this container to work according to the Hanami application conventions as well as your own specific application configuration. However, for advanced cases, you can also directly configure the slice’s container using `prepare_container` on the slice class:
```

Reply:
Thanks for the suggestion here, @jodosha 😄 

Respectfully, I won't apply this one, however, since the dry-rb gem names are not actual code terms, so I've been trying to make sure we never refer to them like that in our documentation and other written materials.